deploy/eck-stack/charts/eck-kibana/examples/ingress/kibana-eks.yaml

```diff
@@ -0,0 +1,44 @@
+# The following is an example of a Kibana resource that is configured to use an Ingress resource in an EKS cluster.
+#
+
+# Name of the Kibana instance.
+#
+fullnameOverride: kibana
+
+spec:
+  # Reference to ECK-managed Elasticsearch instance, ideally from {{ "elasticsearch.fullname" }}
+  #
+  elasticsearchRef:
+    name: elasticsearch
+  config:
+    server:
+      publicBaseUrl: "https://kibana.company.dev"
+
+ingress:
+  enabled: true
+  className: alb
+  annotations:
+    alb.ingress.kubernetes.io/scheme: "internet-facing"
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
+    alb.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    alb.ingress.kubernetes.io/target-type: "ip"
+    # To use an ALB with ECK, you must provide a valid ACM certificate ARN or use certificate discovery:
+    # ref: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/guide/ingress/cert_discovery/
+    #
+    # alb.ingress.kubernetes.io/certificate-arn: "arn:aws:acm:us-east-1:00000000000:certificate/b65be571-8220-4f2e-8cb1-94194535d877"
```

Comment:
why is this commented out if this is required ?

Reply:
An acm certificate is absolutely required. Apparently Discovery can attempt to find it automatically, or you can statically configure it here. Since this isn't a valid Arn, I commented it out. Happy to uncomment it, or add additional documentation to make it clearer. Let me know

Reply:
aaah I see, maybe we can add a note to say this needs to be uncommented out and updated with a valid ARN for the ingress to work correrctly.